src/Processing/DefaultProcessor.php

```diff
@@ -691,12 +698,14 @@ public function processSkippedProperties(
 				$object->$propertyName = $processed;
 				$skippedPropertiesContext->removeInitializedProperty($propertyName);
 			} catch (ValueDoesNotMatch | InvalidData $exception) {
-				$type->overwriteInvalidField($fieldName, $exception->getInvalidType());
+				$type->overwriteInvalidField($fieldName, $exception);
+				// TODO: better throw early?
 			}
 		}
 
 		// If any of fields is invalid, throw error
 		if ($type->hasInvalidFields()) {
+			// TODO: Where to get value?
 			throw InvalidData::create($type);
```

Comment:
I guess it's case for NoValue. Invalid values are already tracked by individual invalid fields